src/ui/services/networkService.ts

```diff
@@ -105,6 +109,12 @@ export class NetworkService extends EventEmitter {
             "value": value 
         });
     }
+    public startGame(){
+        if (!this.currentLobby || !this.currentLobbyConnection) {
```

Comment:
This could be more clear as `if (!(this.currentLobby && this.currentLobbyConnection))`